views/index.jade

```diff
@@ -23,19 +23,21 @@ block content
     div.container
       div.row
         div.col-md-6.whatis
-          h2 Build unstoppable applications
+          h2 Welcome to the Ethereum open source community
```

Comment:
"Welcome to the open source community" is too bland and doesn't communicate anything. It's my opinion that generic phrases like "welcome to our home page" or "click this link" are signs that that particular piece of content shouldn't be there